journal/journal/tests/test_views.py

```diff
@@ -0,0 +1,59 @@
+from journal.views import my_view, compose, entry_detail, edit_entry
+from pyramid.testing import DummyRequest
+
+
+def test_my_view(dbtransaction, new_entry):
+    """Test my_view retunrs a lists entries in the response object."""
+    req = DummyRequest()
+    resp = my_view(req)
+    entries_dict = resp['entries']
+    assert entries_dict[0] == new_entry
+
+
+def test_entry_detail_view(dbtransaction, new_entry):
+    """Test entry_detail returns a list of entries in the reposnse object."""
+    req = DummyRequest()
+    req.matchdict = {'entry_id': new_entry.id}
+    resp_dict = entry_detail(req)
+    assert resp_dict['entry'] == new_entry
```

Comment:
What happens when you try to get to the entry view with an ID that does not belong to an existing entry.